packages/framework/src/Form/Admin/Order/OrderItemFormType.php

```diff
@@ -40,6 +45,37 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
                 'error_bubbling' => true,
             ])
+            ->add('priceWithoutVat', MoneyType::class, [
+                'scale' => 6,
+                'constraints' => [
+                    new Constraints\NotBlank([
+                        'message' => 'Please enter price',
```

Comment:
Please distinguish error messages by the field. Eg. "Please enter unit price excluding VAT"

Reply:
Originally, I didn't want to add many translation messages, but I guess it's worth it. Addressed in f85fd7c4d